modules/seqexec/model/jvm/src/main/scala/seqexec/model/config/SeqexecServerConfiguration.scala

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.model.config
+
+import cats.Eq
+import cats.implicits._
+import org.http4s.Uri
+import scala.concurrent.duration.FiniteDuration
+import shapeless.tag.@@
+
+trait GpiSettings
+trait GhostSettings
+
+final case class SeqexecServerConfiguration(
+                          odb:                     String,
```

Comment:
Can we get some Scaladoc for these parameters? Some are obvious but many aren't. (like this one)